src/sprint-mission1/entity/article.js

```diff
@@ -0,0 +1,55 @@
+export default class Article {
+  #id;
+  #title; // 글 제목
+  #content; // 글 내용
+  #writer; // 작성자
+  _likeCount; // 좋아요 수
+  #createdAt; // 생성 일자
+
+  constructor(
+    id,
+    title,
+    content,
+    createdAt,
+    writer = "알수없음",
+    _likeCount = 0,
```

Comment:
```suggestion
  constructor(
    id,
    title,
    content,
    writer = "알수없음",
    _likeCount = 0,
```

createdAt 같은 경우 매개 변수로 받기 보다는 바로 현재 시간을 할당하는 것이 좋을 것 같습니다.